.eslintrc.js

```diff
@@ -75,7 +75,8 @@ module.exports = {
         'import/prefer-default-export': 0,
         'import/extensions': ['warn', 'never', { jsx: 'always', json: 'always' }],
         'no-sequences': ['warn'],
-        'import/no-unresolved': [2, { ignore: ['@deriv/components', '@deriv/shared'] }],
+        // 'import/no-unresolved': [2, { ignore: ['@deriv/components', '@deriv/shared'] }],
+        'import/no-unresolved': 0,
```

Comment:
i think its better to not disable this. why do you need it?

Reply:
Jim and I were checking options to resolve the ESlint conflicts. Since it is not the scope of this PR I am reverting this.